src/convertToCase/validate.js

```diff
@@ -0,0 +1,27 @@
+function validate(text, textCase) {
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errorsText = {
+    /* eslint-disable max-len */
+    emptyTextValue: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    emptyCaseValue: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    invalidCaseValue: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    /* eslint-enable max-len */
+  };
```

Comment:
Let's move these variables outside your function to avoid re-creating whenever this function is executed.